### PR TITLE
Template cleanup

### PR DESCRIPTION
**Pull request by kaitj**
Reorganizes current directory structure and use of submodules to reduce Heroku slug size.

- Git submodule has been removed, which should reduce slug size significantly
- Templates have been added into a new folder afids-templates
  - Within this folder a sub-folder humans has been added for current templates in the event non-human templates are to be added in the future - this will require code to be rewritten in the future though
- Directory paths and variables have been updated

### CODE REVIEW

File: controller.py [hunk 1]
```diff
@@ -476,7 +474,7 @@ def create_figure():
 flx.assets.associate_asset(__name__, 'https://d3js.org/d3.v4.min.js')
 flx.assets.associate_asset(__name__, 'https://unpkg.com/d3-3d/build/d3-3d.min.js')
 
-with open('./afids-examples/sub-MNI2009cAsym/sub-MNI2009cAsym_afids.fcsv') as MNI:
+with open('./afids-templates/human/sub-MNI2009cAsym_afids.fcsv') as MNI:
```
Comment (tkkuehn) [inline controller.py]: Maybe

```with open(os.path.join(AFIDS_HUMAN_DIR, 'sub-MNI2009cAsym_afids.fcsv')) as MNI:``` 

would be a bit better here?

Comment (kaitj) [reply controller.py]: I think potentially yes. Can't remember what the original intention behind this line of code was.

**Comment by tkkuehn** [comment]
Does this need anything more to be ready for review?

File: controller.py (hunk 1 above)
Comment (tkkuehn) [reply controller.py]: Functionally it would be the same, just a bit less work to change if we change the template directory structure in the future.

**Comment by kaitj** [comment]
Nope, was just waiting for the merge for the CI stuff.

File: controller.py (hunk 1 above)
Comment (kaitj) [reply controller.py]: That's true. I can make the changes and push it to the repo. Do we know what the original function of it was for - default template on validator load? We may need to revisit in the future if/when we add more (non-human) templates.

**Pull request merged**